middlewares/signupValidator.js

```diff
@@ -25,7 +25,12 @@ export default class SignupValidator {
   }
 
   static validateNewPassword(req, res, next) {
-    const { newPassword } = req.body;
+    const { newPassword, currentPassword } = req.body;
+    if (newPassword === null || currentPassword === null) {
+      return res.status(BAD_REQUEST).json({
+        message: "Null values are not allowed!"
+      });
+    }
     const isPassword = isAlphanumeric(newPassword) && newPassword.trim().length >= 8;
```

Comment:
@niomwungeri-fabrice, is there any reason why you're preventing the user from entering the password which includes the symbols?

Reply:
Hi, @Musigwa!
Its good question. I think we need to decide as a team because since the beginning we were only accepting alphanumeric passwords only.
But I agree with you we should allow symbols in our passwords.